content/en/armory-enterprise/release-notes/rn-armory-agent/agent-service/agent-service-v-1-0-1.md

```diff
@@ -0,0 +1,8 @@
+---
+title: v1.0.1 Armory Agent Service (2022-01-10)
+toc_hide: true
+version: 01.00.01
+
+---
+
+Uses podname as agentId.
```

Comment:
```suggestion
## Changes

The Agent Service now uses `agentID` for the pod name. This can help with troubleshooting when trying to associate log messages with pods.
```